### ISSUE TITLE
Could not load library libcudnn_ops_infer.so.8. 

### ISSUE THREAD

**Issue by SheikhGalib** [ufal/whisper_streaming#119]
I have installed everything as it is stated. first, gave. ```pip install librosa soundfile``` then ```pip install faster-whisper```. I have cuda version 11.8 and 4090 nvidia GPU on a linux machine. I also have installed cuDNN for cuda 11 specific on the machine. 
Cloned the whisper-streaming git repository and tested with the command line they told to use
```
python3 whisper_online.py myFile.wav --language en --min-chunk-size 1 > out.txt
```
I kept myFile.wav in the same directory as whisper_online.py which is in whisper_streaming
but this error comes every time.
```
INFO    Audio duration is: 625.77 seconds
INFO    Loading Whisper large-v2 model for en...
INFO    done. It took 1.91 seconds.
Could not load library libcudnn_ops_infer.so.8. Error: libcudnn_ops_infer.so.8: cannot open shared object file: No such file or directory
Aborted (core dumped)
```

**Comment by haraGADygyl** [comment]
I had the same issue and it was caused by the fact that cuDNN (CUDA Deep Neural Network) library wasn't properly linked in my system. cuDNN was installed in a venv in another project`s folder.

I'm using Ubuntu 24.04.

Run this to find there is cuDNN installed:
```
sudo find / -type f -name libcudnn_ops_infer.so.8
```

In my case this outputs: 
```
/home/user/GitHub/Python/test-whisper/.venv/lib/python3.12/site-packages/nvidia/cudnn/lib/libcudnn_ops_infer.so.8
```

You can either install cuDNN in you current project`s folder or run this:
```
echo 'export LD_LIBRARY_PATH=/home/user/GitHub/Python/test-whisper/.venv/lib/python3.12/site-packages/nvidia/cudnn/lib:$LD_LIBRARY_PATH' >> ~/.bashrc
```

Replace `test-whisper` with the folder of you cuDNN instalation

**Comment by Gldkslfmsd** [comment]
> ```
> echo 'export LD_LIBRARY_PATH=/home/user/GitHub/Python/test-whisper/.venv/lib/python3.12/site-packages/nvidia/cudnn/lib:$LD_LIBRARY_PATH' >> ~/.bashrc
> ```


Yes, there's an instruction in faster-whisper README about the LD_LIBRARY_PATH. It's easy to overlook.

**Comment by Nik-Kras** [comment]
Just a note: vast.ai machine was Ubuntu 22.04 x84_64 machine. And I tested it for this machine only.

Here is what helped me:

1. Install NVCC driver
```bash
sudo apt install nvidia-cuda-toolkit
```

2. Install cuDNN (Version 8!)

It is not trivial to install version 8!
If you go to the official website to install cuDNN and enter your machine's details, you will only get the latest version — version 9. I tried it, but it doesn't work (even when you link from calls of version 8 to version 9).

Official [Website](https://developer.nvidia.com/cudnn)

So, you would need to go to [archive](https://developer.nvidia.com/rdp/cudnn-archive) and submit your request for version 8.

After that, you will get a file called `.deb`. In my case, I got `cudnn-local-repo-ubuntu2204-8.9.7.29_1.0-1_amd64.deb`

Here is how you install it:

```bash
sudo dpkg -i cudnn-local-repo-ubuntu2204-8.9.7.29_1.0-1_amd64.deb
sudo cp /var/cudnn-local-repo-ubuntu2204-8.9.7/cudnn-local-*.gpg /usr/share/keyrings/
```

Maybe you would also need to do that:

```bash
sudo apt-get update
sudo apt-get install --reinstall libcudnn8 libcudnn8-dev libcudnn8-samples
```

3. Verify installation

```bash
ls /usr/lib/x86_64-linux-gnu/libcudnn* | grep libcudnn_ops_infer.so.8
ls /usr/lib/x86_64-linux-gnu/libcudnn* | grep libcudnn_cnn_infer.so.8
```
